xdsl/backend/riscv/lowering/riscv_arith_lowering.py

```diff
@@ -128,16 +187,8 @@ def match_and_rewrite(self, op: arith.CeilDivUI, rewriter: PatternRewriter) -> N
         raise NotImplementedError("CeilDivUI is not supported")
 
 
-class LowerArithRemUI(RewritePattern):
-    @op_type_rewrite_pattern
-    def match_and_rewrite(self, op: arith.RemUI, rewriter: PatternRewriter) -> None:
-        rewriter.replace_matched_op(riscv.RemuOp(op.lhs, op.rhs))
-
-
-class LowerArithRemSI(RewritePattern):
-    @op_type_rewrite_pattern
-    def match_and_rewrite(self, op: arith.RemSI, rewriter: PatternRewriter) -> None:
-        rewriter.replace_matched_op([riscv.RemOp(op.lhs, op.rhs)])
+lower_arith_remui = lower_signless_integer_binary_op(arith.RemUI, riscv.RemuOp)
```

Comment:
`nit`: why are these definitions in lowercase while the other ones are in PascalCase ?

Reply:
Ah it's because I forgot to rename them, will do that now